Dockerfile

```diff
@@ -9,11 +11,8 @@ RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o derek .
 
 FROM alpine:3.10 as ship
 
-RUN apk --no-cache add curl ca-certificates \ 
```

Comment:
I would say that we still need "ca-certificates" for accessing Derek URLs over HTTPS.

Reply:
Forgot about that :+1: will do

Reply:
Brought back :+1: